test/unit/range/views/view_async_input_buffer_test.cpp

```diff
@@ -20,67 +20,67 @@
 
 #include "../iterator_test_template.hpp"
 
-using namespace seqan3;
+using seqan3::operator""_dna4;
 
 using iterator_type = std::ranges::iterator_t<
-    decltype(std::declval<std::vector<seqan3::dna4>&>() | views::async_input_buffer(3))>;
+    decltype(std::declval<seqan3::dna4_vector&>() | seqan3::views::async_input_buffer(3))>;
 
 template <>
 struct iterator_fixture<iterator_type> : public ::testing::Test
 {
     using iterator_tag = std::input_iterator_tag;
     static constexpr bool const_iterable = false;
 
-    std::vector<seqan3::dna4> vec{"ACGTACGTACGTATCGAGAGCTTTAGC"_dna4};
```

Comment:
why change these?

Reply:
I thought it's a nicer and shorter style.
I changed it in ```test/unit/range/views/view_convert_test.cpp``` because of the line lenght (otherwise it got too wide for one line), and then I changed it every where else aswell for consistency.